app/controllers/key_pairs_controller.rb

```diff
@@ -60,11 +89,13 @@ def update
 
     respond_to do |format|
       if @key_pair.update_attributes(key_pair_params)
-        format.html { redirect_to @key_pair, notice: 'Key pair was successfully updated.' }
+        format.html { redirect_to @key_pair,
+                                  notice: 'Key pair was successfully updated.'}
         format.json { head :no_content }
       else
         format.html { render action: "edit" }
-        format.json { render json: @key_pair.errors, status: :unprocessable_entity }
+        format.json { render json: @key_pair.errors,
+                                   status: :unprocessable_entity}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 98, 65 should be on its own line.